metricbeat/magefile.go

```diff
@@ -27,17 +27,30 @@ import (
 	"time"
 
 	"github.com/magefile/mage/mg"
-	"github.com/magefile/mage/sh"
 
 	devtools "github.com/elastic/beats/dev-tools/mage"
 	metricbeat "github.com/elastic/beats/metricbeat/scripts/mage"
 
+	// mage:import
+	build "github.com/elastic/beats/dev-tools/mage/target/build"
 	// mage:import
 	"github.com/elastic/beats/dev-tools/mage/target/common"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/dashboards"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/docs"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/pkg"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/test"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/unittest"
+	// mage:import
+	update "github.com/elastic/beats/dev-tools/mage/target/update"
```

Comment:
Oof, that's a many imports. As we have a magefile per project, keeping everything in sync might be a pain. Maybe we should have common target profiles, so to import one mage-package only.

e.g. github.com/elastic/beats/dev-tools/mage/profile/beats